service/src/intTest/java/uk/nhs/adaptors/gp2gp/ehr/EhrExtractStatusServiceIT.java

```diff
@@ -44,24 +50,78 @@ public class EhrExtractStatusServiceIT {
 
     private static final Instant NOW = Instant.now();
     private static final Instant FIVE_DAYS_AGO = NOW.minus(Duration.ofDays(5));
+    private static final Instant TEN_DAYS_AGO = NOW.minus(Duration.ofDays(10));
     private static final int DEFAULT_CONTENT_LENGTH = 244;
     private static final String CONTENT_TYPE_MSWORD = "application/msword";
     public static final String JSON_SUFFIX = ".json";
+    private static final ErrorConstants ACK_TIMEOUT_ERROR = ErrorConstants.ACK_TIMEOUT;
 
     @Autowired
     private EhrExtractStatusService ehrExtractStatusService;
 
     @Autowired
     private EhrExtractStatusRepository ehrExtractStatusRepository;
 
+    @Autowired
+    private EhrExtractTimeoutScheduler ehrExtractTimeoutScheduler;
+
     @MockBean
     private TimestampService timestampService;
 
+    @MockBean
+    private Logger logger;
+
     @BeforeEach
     public void emptyDatabase() {
         ehrExtractStatusRepository.deleteAll();
     }
 
+    @Test
+    void updateEhrExtractStatusAckTest() {
```

Comment:
I think this name can be improved:

* What is the tests objective?

Can we do something like `When_With_Except` here?